addons/website_slides/views/website_slides_templates_lesson.xml

```diff
@@ -98,10 +100,9 @@
 <!-- Slide sub-template: display an item in a list of related slides (Related, Most Viewed, ...) -->
 <template id="slide_aside_card" name="Related Slide">
     <a class="list-group-item list-group-item-action d-flex align-items-start px-2" t-att-href="'/slides/slide/%s' % (slug(aside_slide))">
-        <t t-set="slide_image" t-value="website.image_url(aside_slide, 'image_1024')"/>
+        <t t-set="slide_image" t-value="website.image_url(aside_slide, 'image_256') if can_access_channel or aside_slide.is_preview else default_background_image"/>
 
-        <div t-if="aside_slide.image_1024" class="flex-shrink-0 me-1 border" t-attf-style="width: 20%; padding-top: 20%; background-image: url(#{slide_image}); background-size: cover; background-position:center"/>
-        <div t-else="" class="o_wslides_card_default_background flex-shrink-0 me-1" t-attf-style="width: 20%; padding-top: 20%;"/>
+        <div class="me-1 border" t-attf-style="min-width: 20%; padding-top: 20%; background-image: url(#{slide_image}); background-size: cover; background-position:center"/>
```

Comment:
Same here, a lot of type styling could be in scss and we just keep the dynamic background-image property.